examples/js/loaders/GLTFLoader.js

```diff
@@ -2367,17 +2374,14 @@ THREE.GLTFLoader = ( function () {
 
 		source = json.images[ textureDef.source ];
 
-		var loader;
+		var loader = textureExtensions[ EXTENSIONS.MSFT_TEXTURE_DDS ]
+			? parser.extensions[ EXTENSIONS.MSFT_TEXTURE_DDS ].ddsLoader
+			: this.textureLoader;
```

Comment:
This looks like a mistake during a rebase — support for `MSFT_texture_dds` was removed in https://github.com/mrdoob/three.js/pull/21271 and shouldn't be referenced here.

Reply:
Oh you're right -- thanks for the catch. I just updated it. Mind taking another look? I removed some unused variables in the function (specifically `parser` and `textureExtensions`) and simplified the definition of `source`, too.